CONTRIBUTING.md

```diff
@@ -42,14 +42,15 @@ Releases are somewhat automated. The github action, `publish`, watches for any t
 
 1. Create a new branch like `git checkout -b prepare-0.20.1`
 2. Update all Cargo.toml files to the new version, `version = 0.20.1`
-3. Update all inter-dependent crates, i.e. trust-dns-resolver to use `trust-dns-proto = 0.20.1`
-4. Update [CHANGELOG.md](CHANGELOG.md) to include all PR's (of consequence) since the previous release
-5. Push to Github, create a PR and merge in `main` or the target release branch.
-6. Go to [Releases](https://github.com/bluejekyll/trust-dns/releases) and `Draft a new release`
-7. Give it a `Tag Version` of `vX.x.x`, e.g. `v0.20.1`, *make sure this is tagging the correct branch, e.g. `main` or `release/0.19`*
-8. Give it a  `Release Title` of something key to the release
-9. Copy and pase the part of the CHANGELOG.md for this release into `Describe this release`
-10. `Publish Release`, this will kick off the publish workflow
+3. Update dependencies, `cargo update`
```

Comment:
BTW, I think Markdown has a way of automatically numbering these, to prevent you from getting diffs like these? I think you can just use `1.` for every item and it will fix the numbering for you.